### PR TITLE
Add a data node list component

### LINKED ISSUE #317: Add node list component
Add a Nodes component that can output a list of nodes for the current network.

# Data source
#304 added a commited `vaguer.[NETWORK].json`. We can use this, or alternatively, that build script already downloads the network.toml, store that output and use that.

# Page
https://docs.vega.xyz/docs/mainnet/api/overview

### PR DESCRIPTION
**Pull request by edd**
Add build time, the network.toml for testnet and mainnet is fetched, along with a vaguer
run. This can then be output by a new component

- [x] Update build-post-openapi to save network.toml as json
- [x] Generate node list for api overview page
- [x] Sort nodes by Vaguer output, then alphabetically
- [x] Make mainnetNetworkToml and testnetNetworkToml package.json config vars 
- [x] Make a link component for the tomls
- [x] Update `tools/vega-wallet/cli-wallet/latest/guides/manage-networks` so that this won't happen again
- [x] Get opinions on content and rejig

Closes #317

### CODE REVIEW

**Comment by edd** [comment]
Currently:

Screenshot 2022-10-06 at 15 04 42

Screenshot 2022-10-06 at 15 05 37

**Availability** isn't a great summary. What it is is that https://github.com/vegaprotocol/vaguer, when the docs were build, saw that this node returned the same GraphQL result as a representative set of nodes. It's not a guarantee of availability, or speed, or even correctness, but it is ... something.

File: docs/api/overview.md [hunk 1]
```diff
@@ -12,6 +16,8 @@ As most of the APIs are designed to use for trading-related queries, the best pl
 [Vega Fairground documentation](https://docs.fairground.vega.xyz/) includes [how-to guides](https://docs.fairground.vega.xyz/docs/api-howtos/) and a Gitpod environment to try out Vega Fairground. 
 :::
```
Comment (candida-d) [inline docs/api/overview.md:18]:
```suggestion
## Node API endpoints
```

File: versioned_docs/version-v0.53/api/overview.md [hunk 2]
```diff
@@ -12,6 +16,8 @@ As most of the APIs are designed to use for trading-related queries, the best pl
 [Vega Fairground documentation](https://docs.fairground.vega.xyz/) includes [how-to guides](https://docs.fairground.vega.xyz/docs/api-howtos/) and a Gitpod environment to try out Vega Fairground. 
 :::
```
Comment (candida-d) [inline versioned_docs/version-v0.53/api/overview.md:18]:
```suggestion
## Node API endpoints
```

File: src/components/DataNodes.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+function TableForNodes(listOfNodes, vaguerResult) {
+    const listOfNodesPlusVaguer = vaguerList(listOfNodes, vaguerResult)
+
+    return (<table>
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Address</th>
+          <th>Availability</th>
```
Comment (candida-d) [inline src/components/DataNodes.js:52]:
```suggestion
          <th>Connectivity</th>
```

File: src/components/DataNodes.js [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+  const source = isMainnet ? 'https://github.com/vegaprotocol/networks' : 'https://github.com/vegaprotocol/networks-internal'
+
+  if (!addresses) {
+    throw new Error("Could not load addresses");
+  }
+
+  return (
+    <div className="DataNodeList">
+        <Tabs>
+            <TabItem value="grpc" label="GRPC endpoints">{TableForNodes(addresses.GRPC.Hosts, vaguer)}</TabItem>
```
Comment (candida-d) [inline src/components/DataNodes.js:99]:
```suggestion
            <TabItem value="grpc" label="GRPC">{TableForNodes(addresses.GRPC.Hosts, vaguer)}</TabItem>
```

File: src/components/DataNodes.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+            <TabItem value="graphql" label="GraphQL endpoints">{TableForNodes(addresses.GraphQL.Hosts, vaguer)}</TabItem>
```
Comment (candida-d) [inline src/components/DataNodes.js:100]:
```suggestion
            <TabItem value="graphql" label="GraphQL">{TableForNodes(addresses.GraphQL.Hosts, vaguer)}</TabItem>
```

**Review by candida-d — APPROVED** [review]

**Pull request merged**